Mage/RPCClient.cs

```diff
@@ -33,7 +33,7 @@ public class RPCClient : JSONRPC {
 		logger.verbose("[" + methodName + "] send to remote");
 
 		// Make the RPC call
-		this.call (methodName, parameters, headers, (Exception error, JObject responseObject) => {
+		this.call (1, methodName, parameters, headers, (Exception error, JObject responseObject) => {
```

Comment:
this is an absolute necessity to the JSON RPC library, according to specs, NULL ID can only be given in case of notification, however no response would be given back.

http://www.jsonrpc.org/specification#request_object